internal/powerplatform/customtypes/uuid.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
if for tenant settings we are starting to use custom uuid type instead of string, we should do it for other resources as well. Is the value added of having custom type worth the refactoring?